content/working_with/console/customization/overrides.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bt_wiki
+title: Labels Overrides
+category: Console
+draft: false
+weight: 180
+---
+
+You can override a number of texts and labels used in {{< param cfy_console_name >}} by supplying a file containing replacements for default texts or labels.
+
+To do that:
+
+1. Create `/opt/cloudify-stage/dist/userData/overrides.json` file.
+1. Go to https://github.com/cloudify-cosmo/cloudify-stage/blob/<version>-build/app/translations/en.json (replace `<version>` with your product version, for example `5.1.0`), choose configurations parts to modify (you can omit all unmodified parts to use default values) and save them to the created file.
```

Comment:
We should probably add a sample of a few lines from that file.

Reply:
Done